web/client/components/import/dragZone/enhancers/__tests__/useFiles-test.js

```diff
@@ -57,6 +57,45 @@ describe('useFiles enhancer', () => {
         expect(spyLoadAnnotations).toNotHaveBeenCalled();
         expect(spySetLayers).toNotHaveBeenCalled();
     });
+    it('useFiles rendering with map with pre-existing mapId', (done) => {
+        const actions = {
+            loadMap: (conf, mapId, zoomToExtent ) => {
+                expect(conf).toExist();
+                expect(conf.map).toExist();
+                expect(conf.map.bbox).toExist();
+                expect(conf.map.center).toExist();
+                expect(conf.map.zoom).toExist();
+                expect(mapId).toExist();
+                expect(zoomToExtent).toBe(false);
+                done();
+            },
+            loadMapInfo: (mapId, info) => {
+                expect(mapId).toBe("10");
```

Comment:
are you sure this check are done?

because when the done() in loadMap is called the test stops, so please make sure they are called for example put a 
```js 
expect(false).toBeTruthy();
```

in this way you are sure that the test would fail, and maybe we can move here the call to done() 
also when you move here the done add the same tricky check on the loadMap callback to see if everything works as expected

Reply:
I modified the test and checked the coverage, this now looks like it is executed